CPS-?/README.md

```diff
@@ -0,0 +1,170 @@
+
+### #2: Emergency group to support the disaster recovery plan.
+  -attacks on persons if known
+  -we don’t want attackers know who that group is?
+
+Recommendation:
+
+### #3: Emergency change that needs to be done quickly.
+  -a nuclear option
+  -change requires “just do it” 
+  -a security that fix must be implemented but CANNOT be communicated publicly
+
+Recommendation:
+
+### #4: Operational Security (OPSEC) of the committee.
+  -physical security
+  -digital security
+
+Recommendation:
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+Recommendation:
+
+### #6: DRep keys lost or stolen or DRep keys sold to a bad actor.
+
+Recommendation:
+
+### #7: Lack of governance for an extended period of time.
+  -Uncontrolled hard forks by SPOs
+  -Risk of no confidence
+
+Recommendation:
+
+### #8: Lack of trust in voter tooling.
+
+Recommendation:
+
+### #9: Trust in the wallets.
+  -wallet voting attack vector
+  -hardware wallet support
+
+Recommendation:
+
+
+### #10: Corruption - in any form, any number of actors.
+  -Code of Conduct mitigation
+  -on-chain mitigation
+
+
+Recommendation:
+
+### #11: Collusion - the bad version of collaboration to do something illegal or undesirable.
+
+Recommendation:
+
+### #12: Vote buying - Ada kickbacks.
+
+Recommendation:
+
+### #13: IGCO - initial governance coin offering
+  -offering voters a token of unknown value by a representative to acquire delegators
+  -including tokens other than Ada
+
+Recommendation:
+
+### #14: Powerful Voter Collusion
+  -demotivates smaller bag holders
+  -cause a loss of participation
+  -too many voters abstain
+  -minority controls the outcome
+  -can push through any proposal
+
+Recommendation:
+
+### #15: Persistence - Removal of proposals by attackers.
+
+Recommendation:
+
+### #16: Risk of Proposal overload - Risk of DDOS.
+
+Recommendation:
+
+### #17: Excessive actions.
+  -Drawing too much money
+  -Changing parameters that break the chain (block size=0 for example)
+  -overlapping actions and duplicate actions
+
+Recommendation:
+
+### #18: Lack of technical understanding by DReps.
+Education of correct procedures for DReps
+
+Recommendation:
+
+### #19: Sybil behavior - one person acting as multiple DReps.
+
+Recommendation:
+
+### #20: Overpower Risk - One DRep or entity with too much power.
+
+Recommendation:
+
+### #21: Superstar attack - one “superstar” person that everybody delegates to becomes a dictator with little skin in the game.
+(Example: Elon Musk as a DRep)
+
+### #22: High chain load impacting governance.
+
+Recommendation:
+
+### #23: Unknown external regulation risks - affecting a specific DRep by country or affecting governance as a whole.
+
+Recommendation:
+
```

Comment:
- Form a Regulation Preparedness & Action Committee
- All issues of importance can be brought to the floor of the committee for evaluation, discussion, recommendation and/or action

Reply:
Maybe I need to break this into 2 types of risks.
1. Indvidual DRep.
2. Cardano Governance process as a whole.

1. I was thinking of just making it the responsibility of each DRep to comply with their local laws. Otherwise, legal fees would skyrocket. If a DRep from a particular region runs into legal issues with acting as a DRep they can either fight it or resign.

2. For Cardano governance process having legal issues, kick that over to the Cardano Foundation to fight.

What do you think?

Reply:
Yes, I agree. Engaging with the CF for Governance legal challenges is probably the best course of action. At least initially, to get things aligned. It may be worth having a point of contact identified at CF as a resource for individual DReps who face legal challenges. If for nothing but some general guidance. Just having someone to talk to who understands the situation can be a huge help just for the stress factor alone.